main.rkt

```diff
@@ -28,3 +42,28 @@
     [(_ x:id e)
      (printf "MACRO WORKED\n")
      #'(define x e)]))
+
+;; =============================================================================
+;;
+;; Constants
+
+(define π pi)
+(define π/2 (/ π 2))
+(define π/3 (/ π 3))
+(define π/4 (/ π 4))
+(define π/6 (/ π 6))
+(define π/12 (/ π 12))
+(define 2/π (/ 2 π))
+(define 3π/4 (* 3 π/4))
+
+(begin-for-syntax
+ (define (find-machine-ε ε)
+  (if (= 1.0 (+ ε 1.0))
+      (* 2 ε)
+      (find-machine-ε (/ ε 2)))))
+
+(define-syntax (machine-ε₀ stx)
+  #`#,(find-machine-ε 1.0))
+
+(define machine-ε (machine-ε₀))
+(define *machine-epsilon* machine-ε)
```

Comment:
Does that compile for you? That was my first attempt, but it kept complaining about referencing an undefined variable. That is why I made that random macro that wraps the function call. Does this work for you:
```racket
#lang racket
(provide a b)
(define a 1)
(define b (+ a 1))
```
This is an error for me. It complains that `b` references an uninitialized variable.